src/ChooseThemeComponent/ThemeOption.js

```diff
@@ -0,0 +1,43 @@
+import Rome from "./assets/Rome.jpg";
+import ArrowL from"./assets/ArrowL.png";
+import ArrowR from"./assets/ArrowR.png";
+import './ChooseTheme.css';
+
+
+
+function ThemeOption(){
+    // <div class="rome-container">
+
+    //     <img src={process.env.PUBLIC_URL+Rome} class="option"/>
+    
+    //     {/* <div>
+    //     <button class="option-btn"><p>Rome</p></button>
+    //     </div> */}
+    // </div>
+    return (
+        <div class="main-container">
+
+            <div>
+            <button class="arrowL"><img src={process.env.PUBLIC_URL+ArrowL} class="arrowLImg"onMouseOver/></button>
+            </div>
+
+            {/* <Carousel>
+                <Item>HI</Item>
+            </Carousel> */}
+
+            <div class="rome-container">
+
+                <img src={process.env.PUBLIC_URL+Rome} class="option"/>
+                
+                <div>
+                <button class="option-btn"><p>Rome</p></button>
```

Comment:
I'm not too sure what you mean... what should I do instead?

Reply:
sorry, the GitHub website deleted portions of the text I was referring to my bad.  

I meant to say

"Minor issue, but for short texts like here "Rome" it's preferred to use a span tag instead of a p tag
for proper semantics purposes."